### ISSUE TITLE
Training with single channel image negative recall

### ISSUE THREAD

**Issue by hatefuleight** [open-mmlab/mmdetection#3839]
I solved it, thanks.

**Comment by ZwwWayne** [comment]
The pre-trained model is trained on ImageNet, which is a dataset containing RGB images. However, BN's mean and variance are fixed during training. I am not sure whether this is an issue, but this behavior seems to be risky.

**Comment by shinya7y** [comment]
`load_from = None`
It will be helpful to:
(1) fine-tune Faster R-CNN from a COCO pre-trained model (to reveal the issue)
(2) fine-tune Mask R-CNN from a COCO pre-trained model

`lr=1e-06` may be too low for training from ImageNet pre-trained models.
`frozen_stages=1` might harm accuracy for non-RGB inputs.
Changing `Normalize` may improve accuracy (probably minor issue).

**Comment by efedoganay** [comment]
> The pre-trained model is trained on ImageNet, which is a dataset containing RGB images. However, BN's mean and variance are fixed during training. I am not sure whether this is an issue, but this behavior seems to be risky.

I guess it is possible to train without pretrained backbone so I changed `pretrained= None` but still getting the same result.

**Comment by ZwwWayne** [comment]
Hi @efedoganay ,
The normalization of your images might also have problems. Also, if you set `pretrained=None`, you also need to set `norm_eval=False`. You may set break points in the model to check whether the value of tensors are what you expected.

**Comment by efedoganay** [comment]
> Hi @efedoganay ,
> The normalization of your images might also have problems. Also, if you set `pretrained=None`, you also need to set `norm_eval=False`. You may set break points in the model to check whether the value of tensors are what you expected.

Hi, I also changed norm_eval to False. The problem remains and here are the results of the first epoch:

2020-10-04 01:49:57,302 - mmdet - INFO - Epoch [1][50/309]	lr: 9.890e-06, eta: 5:42:36, time: 0.333, data_time: 0.055, memory: 2131, loss_rpn_cls: 0.6938, loss_rpn_bbox: 0.0653, loss_cls: 0.8597, acc: 65.4570, loss_bbox: 0.0146, loss_mask: 5.0028, loss: 6.6361
2020-10-04 01:50:11,656 - mmdet - INFO - Epoch [1][100/309]	lr: 1.988e-05, eta: 5:18:21, time: 0.286, data_time: 0.010, memory: 2131, loss_rpn_cls: 0.6855, loss_rpn_bbox: 0.0481, loss_cls: 0.1833, acc: 99.0898, loss_bbox: 0.0099, loss_mask: 1.6308, loss: 2.5577
2020-10-04 01:50:26,111 - mmdet - INFO - Epoch [1][150/309]	lr: 2.987e-05, eta: 5:11:05, time: 0.289, data_time: 0.011, memory: 2131, loss_rpn_cls: 0.6706, loss_rpn_bbox: 0.0488, loss_cls: 0.1764, acc: 98.8125, loss_bbox: 0.0158, loss_mask: 1.4170, loss: 2.3285
2020-10-04 01:50:40,397 - mmdet - INFO - Epoch [1][200/309]	lr: 3.986e-05, eta: 5:06:27, time: 0.286, data_time: 0.011, memory: 2131, loss_rpn_cls: 0.6495, loss_rpn_bbox: 0.0488, loss_cls: 0.1505, acc: 99.0039, loss_bbox: 0.0109, loss_mask: 1.0659, loss: 1.9255
2020-10-04 01:50:54,708 - mmdet - INFO - Epoch [1][250/309]	lr: 4.985e-05, eta: 5:03:41, time: 0.286, data_time: 0.011, memory: 2131, loss_rpn_cls: 0.6214, loss_rpn_bbox: 0.0448, loss_cls: 0.1775, acc: 99.0625, loss_bbox: 0.0066, loss_mask: 0.8509, loss: 1.7011
2020-10-04 01:51:09,137 - mmdet - INFO - Epoch [1][300/309]	lr: 5.984e-05, eta: 5:02:10, time: 0.289, data_time: 0.012, memory: 2131, loss_rpn_cls: 0.5871, loss_rpn_bbox: 0.0495, loss_cls: 0.1941, acc: 99.0000, loss_bbox: 0.0029, loss_mask: 0.6309, loss: 1.4645

I think my network is able to learn mask and classification but rpn_bbox and loss_bbox are extremely small compared to them
